Sources/AsyncQueue/FIFOQueue.swift

```diff
@@ -94,7 +94,7 @@ public final class FIFOQueue: Sendable {
     }
 
     /// Schedules an asynchronous throwing task and returns after the task is complete.
-    /// The scheduled task will not execute until all prior tasks have completed.
+    /// The scheduled task will not execute until all prior tasks – including suspended tasks – have completed.
```

Comment:
I'm pretty sure we missed one https://github.com/dfed/swift-async-queue/blob/c4d455f4e9d8116e8e9479e35e3400f2f3f94734/Sources/AsyncQueue/FIFOQueue.swift#L113

Reply:
ahh I missed all the methods added in #7. cdfdfe6